status/versionInfo.json

```
@@ -1,15 +1,15 @@
 {
   "min_app_version": {
-    "ios": "2.30.0.2",
-    "android": "2.30.0.2"
+    "ios": "2.35.0.1",
+    "android": "2.35.0.1"
   },
   "min_app_version_pagopa": {
     "ios": "0.0.0",
     "android": "0.0.0"
   },
   "latest_released_app_version" : {
-    "ios": "2.32.0.1",
-    "android": "2.32.0.1"
+    "ios": "2.36.0.2",
+    "android": "2.36.0.2"
```

shouldn't this be `2.38.0.1` ?
```suggestion
    "ios": "2.38.0.1",
    "android": "2.38.0.1"
```

---

It is not live yet, that is why I chose to pick a version that is currently available for both systems. What do you think?